clang/docs/ReleaseNotes.rst

```diff
@@ -201,6 +201,8 @@ Bug Fixes to C++ Support
   parameter where we did an incorrect specialization of the initialization of
   the default parameter.
   Fixes (`#68490 <https://github.com/llvm/llvm-project/issues/68490>`_)
+- Fix regression due missing ambiguity check before checking lookup access.
+  Fixes (`80435 <https://github.com/llvm/llvm-project/issues/80435>`_)
```

Comment:
I think we should remove the changelog entry and backport instead @AaronBallman @erichkeane

Reply:
Based on offline conversation I will remove the release note.